packages/sakuli-legacy/src/loader/legacy-project-properties.class.ts

```diff
@@ -164,6 +164,17 @@ export class LegacyProjectProperties {
     );
   }
 
+  /**
+   * Disables the automatic iteration of frames.
+   * If this is enabled, switching frames needs to be done manually via the webdriver.
+   * For further information click [here](https://sakuli.io/docs/cookbook/frames/).
+   *
+   *
+   * Default value: false
+   */
+  @BooleanProperty("testsuite.manualFrameSwitch")
+  manualFrames: boolean = false;
```

Comment:
I think `manualFrameSwitch` could be a misleading name for a toggle that *"Disables the automatic iteration of frames"*. Especially that this is an opt out feature is not reflected properly, in my opinion. 
I'd suggest something like:
- `disableAutoFrameSwitch`
- `disableSearchThroughFrames`
- `disableSearchInFrames`